src/master-playlist-controller.js

```diff
@@ -820,14 +820,13 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     if (!buffered.length) {
       // return true if the playhead reached the absolute end of the playlist
-      return absolutePlaylistEnd - currentTime <= Ranges.TIME_FUDGE_FACTOR;
+      return absolutePlaylistEnd - currentTime <= 0.1;
```

Comment:
Since it is used in multiple places, may be worth creating a variable like `Ranges.SAFE_END_DELTA` and putting the comment there.